static/app/components/forms/controls/rangeSlider/index.tsx

```diff
@@ -219,6 +221,12 @@ function RangeSlider({
   );
 }
 
+const StyledSlider = styled(Slider, {
+  shouldForwardProp: prop => isPropValid(prop),
```

Comment:
this is the default behaviour of emotion and shouldn’t be necessary.

Reply:
Ah yes, I need to swap the forwardRef and styled order inside Slider though, the inline definition otherwise seems to receive all the props